auto_nag/scripts/configs/tools.json

```diff
@@ -657,5 +657,9 @@
             "jkratzer@mozilla.com"
         ],
         "max_ni": 1
+    },
+    "good_first_bug_unassign_inactive": {
```

Comment:
you must add an entry `receivers` for the email: with you of course to analyze the results and anybody else who's interested to have a report.

Reply:
Please add sylvestre as receiver (I've his agreement).